Automattic/newspack-blocks
# Add French (Belgium) translation to the blocks

### All Submissions:






### Changes proposed in this Pull Request:

This PR adds fr_BE translation (tbh it's the same as French fr_FR in the future...).

### How to test the changes in this Pull Request:

1. Switch your site language to "Français de Belgique" (you don't need to switch your wp-admin interface to fr_BE unless you want to test the translation for the editor)
2. Make sure you have a page with the homepage block post, check the front-end and see if the author is translated ("by" should be "par")

### Other information:
